ffcx/codegeneration/expression_generator.py

```diff
@@ -61,10 +61,16 @@ def generate(self):
 
     def generate_geometry_tables(self):
         """Generate static tables of geometry data."""
-        # Currently we only support circumradius
         ufl_geometry = {
+            ufl.geometry.FacetEdgeVectors: "facet_edge_vertices",
```

Comment:
Ugh, the naming (unrelated to the PR) is a bit inconsistent with UFL. Nvm, can you add a quick test so we're sure these are covered for the future?

Reply:
Yep, I'll try to add a test sometime this week.

Reply:
@michalhabera what is the best way to test these? It is a pain to derive all these expressions by hand.

As 99 % of these expressions rely on basix, alot of tests will just check that we call basix in a consistent fashion.
Is this sufficient?